modules/extensions/galasa-extensions-parent/dev.galasa.ras.couchdb/src/main/java/dev/galasa/ras/couchdb/internal/CouchdbDirectoryService.java

```diff
@@ -554,8 +548,40 @@ public IRunResult getRunById(@NotNull String runId) throws ResultArchiveStoreExc
         try {
             return fetchRun(runId);
         } catch (Exception e) {
-            return null;  // This runid may not belong to this RAS, so ignore all errors
+            return null; // This runid may not belong to this RAS, so ignore all errors
         }
     }
-}
 
+    private List<IRunResult> getRunsByCritera(String viewName, String criteriaValue) throws ResultArchiveStoreException{
```

Comment:
Missing an `i` in criteria - I think something like `getRunsFromViewByKey` would be more descriptive than criteria